### DISCUSSION TITLE
Copilot not showing completion while ctrl+enter works well

### DISCUSSION THREAD

**Discussion by cuiyeshuai** [community/community#11224] [Copilot Conversations] [answered]
output is attached below.
when using copilot without ctrl+enter, the output of copilot always ends with [ghostText] Breaking, no choices.
<img width="1860" alt="截屏2022-02-09 下午3 58 48" src="https://user-images.githubusercontent.com/61742973/153147460-6ff59303-e605-4c25-82df-6fff3b354b37.png">

`
  [INFO] [default] [2022-02-09T07:48:53.971Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-aatest
  [INFO] [default] [2022-02-09T07:48:55.859Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-aatest/completions] took 1887 ms
  [INFO] [streamChoices] [2022-02-09T07:48:55.863Z] solution 2 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc0cjOPGvmLSapO8jz33k3KT1Glh}] created: [{1644392934}]
  [INFO] [streamChoices] [2022-02-09T07:48:55.864Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc0cjOPGvmLSapO8jz33k3KT1Glh}] created: [{1644392934}]
  [INFO] [streamChoices] [2022-02-09T07:48:55.865Z] solution 1 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc0cjOPGvmLSapO8jz33k3KT1Glh}] created: [{1644392934}]
  [INFO] [ghostText] [2022-02-09T07:48:55.866Z] Breaking, no choices
  
[INFO] [solutions] [2022-02-09T07:49:42.176Z] prompt: {"prefix":"// Language: javascript\n// Path: copilot.js\nfunction calculateDaysBetweenDates(begin, end) {\n","suffix":""}
  [INFO] [default] [2022-02-09T07:49:42.177Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-aatest
  [INFO] [default] [2022-02-09T07:49:50.544Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-aatest/completions] took 8367 ms
  [INFO] [streamChoices] [2022-02-09T07:49:50.547Z] solution 7 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc1OkH70RLGdwgI7be2yLe9AzzRP}] created: [{1644392982}]
  [INFO] [streamChoices] [2022-02-09T07:49:50.548Z] solution 8 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc1OkH70RLGdwgI7be2yLe9AzzRP}] created: [{1644392982}]
  [INFO] [streamChoices] [2022-02-09T07:49:50.549Z] solution 1 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc1OkH70RLGdwgI7be2yLe9AzzRP}] created: [{1644392982}]
  [INFO] [streamChoices] [2022-02-09T07:49:50.550Z] solution 3 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc1OkH70RLGdwgI7be2yLe9AzzRP}] created: [{1644392982}]
  [INFO] [streamChoices] [2022-02-09T07:49:50.551Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc1OkH70RLGdwgI7be2yLe9AzzRP}] created: [{1644392982}]
  [INFO] [streamChoices] [2022-02-09T07:49:50.949Z] solution 6 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc1OkH70RLGdwgI7be2yLe9AzzRP}] created: [{1644392982}]
  [INFO] [streamChoices] [2022-02-09T07:49:50.952Z] solution 2 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc1OkH70RLGdwgI7be2yLe9AzzRP}] created: [{1644392982}]
  [INFO] [streamChoices] [2022-02-09T07:49:50.953Z] solution 4 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4Zc1OkH70RLGdwgI7be2yLe9AzzRP}] created: [{1644392982}]
  [ERROR] [streamChoices] [2022-02-09T07:49:52.487Z] Error parsing JSON stream data,{"error":{"message":"request timeout"}}
  [INFO] [streamChoices] [2022-02-09T07:49:52.488Z] solution 5 returned. finish reason: [Iteration Done] finishOffset: [undefined] completionId: [{cmpl-4Zc1OkH70RLGdwgI7be2yLe9AzzRP}] created: [{1644392982}]
  [INFO] [solutions] [2022-02-09T07:49:52.488Z] Open Copilot completion: [  var date1 = new Date(begin);
    var date2 = new Date(end]
  [INFO] [streamChoices] [2022-02-09T07:49:52.489Z] solution 9 returned. finish reason: [Iteration Done] finishOffset: [undefined] completionId: [{cmpl-4Zc1OkH70RLGdwgI7be2yLe9AzzRP}] created: [{1644392982}]
  [INFO] [solutions] [2022-02-09T07:49:52.490Z] Open Copilot completion: [  var oneDay = 24*60*60*1000
    var firstDate = new Date]
`

**Comment by davecheney** [comment, marked as answer]
I’m sorry you’re having difficulties. I believe copilot is working, perhaps you could try some other examples, or try copilot on a codebase that you’re working on.

**Reply by cuiyeshuai** [reply]
One suspicion is about proxy, when the proxy is enabled, Copilot always gave the output above and never shows the completion. When the proxy is disabled, the completion might appear. However, what's the reason that causes the difference with or without ctrl+enter, when the proxy is enabled.

**Reply by cuiyeshuai** [reply]
This happens in both version 1.7.4924 and 1.7.4812, the only difference is output of latest version doesn't have this line in output`[INFO] [default] [2022-02-09T08:20:26.535Z] prompt: {"prefix":"// Language: javascript\n// Path: copilot.js\n// find all images without alternate text\n// and give them a red b","suffix":""}`

**Reply by davecheney** [reply]
If there was a proxy that was preventing you talking to the copilot serves you’d see connecting disconnects and timeouts. I’m not seeing that in the output you provided so I believe there are no connectivity issues at the moment. 

I encourage you to try copilot on a codebase you’re working on and see how it goes.

**Reply by cuiyeshuai** [reply]
Thank you for your advice! I have tried copilot in my own codebase and it works well. However, what may be the reason that causes the difference at the start?

**Reply by rogerb831** [reply]
I have this same issue except not even CTRL+Enter works. Works in one codebase, not the another. I'd really like it to work everywhere. Any thoughts why this happens?

**Reply by davecheney** [reply]
@rogerb831 please open a new discussion item. Explain what you did, what you were expecting, and what you saw instead.
